client/src/components/layouts/default.tsx

```diff
@@ -142,18 +148,19 @@ class DefaultLayout extends Component<DefaultLayoutProps> {
       block,
       superBlock,
       t,
-      theme = 'default',
-      user,
-      useTheme = true
+      theme = 'system',
```

Comment:
Although this works because the file is JavaScript in disguiss

There is no type for `system` in the enum, I have tested it with `default` and it gave the same result. I would say keep `default` until the file turned into TypeScript, and we have types that test these values

```suggestion
      theme = 'default',
```

Here is the enum that I mentioned before

```ts
export enum Themes {
  Night = 'night',
  Default = 'default'
}
```

Reply:
Yeah, makes sense to me. When I was first working on this problem and learning the source code, I assumed `default` referred to light mode. The change should work with `default` instead of `system`, and I just tested it out incase.